snippets/native-platforms/ionic2/setup.md

```diff
@@ -1,6 +0,0 @@
-```ts
```

Comment:
Ah gotcha. I just needed to replace them with new content. I can rename the markdown files that contain these steps to `setup` and `use` if you like.